network/counter.js

```diff
@@ -46,7 +46,10 @@ class PacketCounter {
         result[value.src] = value;
       }
     }
+
     this.onTick(result);
+
+    for (var value of this.sessionMap.values()) value.num_bytes = 0;
```

Comment:
Can you actually do the following:

```
this.sessionMap.values().forEach(v => v.num_bytes = 0);
```
That is, can you please use a `forEach` (avoids issues with variable hoisting)?